2021-07-26 17:41:30,384 INFO    MainThread:2767 [wandb_setup.py:_flush():69] setting env: {}
2021-07-26 17:41:30,384 INFO    MainThread:2767 [wandb_setup.py:_flush():69] setting login settings: {}
2021-07-26 17:41:30,384 INFO    MainThread:2767 [wandb_setup.py:_flush():69] setting login settings: {}
2021-07-26 17:41:30,384 INFO    MainThread:2767 [wandb_init.py:_log_setup():337] Logging user logs to /home/parsellsx/berkeley-seti/wandb/run-20210726_174130-3idjflv1/logs/debug.log
2021-07-26 17:41:30,384 INFO    MainThread:2767 [wandb_init.py:_log_setup():338] Logging internal logs to /home/parsellsx/berkeley-seti/wandb/run-20210726_174130-3idjflv1/logs/debug-internal.log
2021-07-26 17:41:30,385 INFO    MainThread:2767 [wandb_init.py:_jupyter_setup():288] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f17dee83e10>
2021-07-26 17:41:30,385 INFO    MainThread:2767 [wandb_init.py:init():370] calling init triggers
2021-07-26 17:41:30,385 INFO    MainThread:2767 [wandb_init.py:init():377] wandb.init called with sweep_config: {}
config: {}
2021-07-26 17:41:30,385 INFO    MainThread:2767 [wandb_init.py:init():419] starting backend
2021-07-26 17:41:30,385 INFO    MainThread:2767 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-07-26 17:41:30,398 INFO    MainThread:2767 [backend.py:ensure_launched():135] starting backend process...
2021-07-26 17:41:30,413 INFO    MainThread:2767 [backend.py:ensure_launched():140] started backend process with pid: 3457
2021-07-26 17:41:30,427 INFO    MainThread:2767 [wandb_init.py:init():424] backend started and connected
2021-07-26 17:41:30,576 INFO    MainThread:2767 [wandb_run.py:_label_probe_notebook():844] probe notebook
2021-07-26 17:41:30,577 INFO    MainThread:2767 [wandb_run.py:_label_probe_notebook():851] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2021-07-26 17:41:30,577 INFO    MainThread:2767 [wandb_init.py:init():472] updated telemetry
2021-07-26 17:41:30,577 INFO    MainThread:2767 [wandb_init.py:init():491] communicating current version
2021-07-26 17:41:31,155 INFO    MainThread:2767 [wandb_init.py:init():496] got version response 
2021-07-26 17:41:31,156 INFO    MainThread:2767 [wandb_init.py:init():504] communicating run to backend with 30 second timeout
2021-07-26 17:41:31,277 INFO    MainThread:2767 [wandb_init.py:init():529] starting run threads in backend
2021-07-26 17:41:36,288 INFO    MainThread:2767 [wandb_run.py:_console_start():1623] atexit reg
2021-07-26 17:41:36,288 INFO    MainThread:2767 [wandb_run.py:_redirect():1497] redirect: SettingsConsole.WRAP
2021-07-26 17:41:36,289 INFO    MainThread:2767 [wandb_run.py:_redirect():1534] Wrapping output streams.
2021-07-26 17:41:36,291 INFO    MainThread:2767 [wandb_run.py:_redirect():1558] Redirects installed.
2021-07-26 17:41:36,291 INFO    MainThread:2767 [wandb_init.py:init():554] run started, returning control to user process
2021-07-26 17:41:36,297 INFO    MainThread:2767 [wandb_init.py:_pause_backend():250] pausing backend
2021-07-26 17:41:36,297 INFO    MainThread:2767 [jupyter.py:save_ipynb():284] not saving jupyter notebook
2021-07-26 17:47:43,468 INFO    MainThread:2767 [wandb_init.py:_resume_backend():259] resuming backend
2021-07-26 17:47:43,469 INFO    MainThread:2767 [wandb_init.py:_pause_backend():250] pausing backend
2021-07-26 17:47:43,469 INFO    MainThread:2767 [jupyter.py:save_ipynb():284] not saving jupyter notebook
2021-07-26 17:47:45,410 INFO    MainThread:2767 [wandb_init.py:_resume_backend():259] resuming backend
2021-07-26 17:47:45,411 INFO    MainThread:2767 [wandb_init.py:_pause_backend():250] pausing backend
2021-07-26 17:47:45,411 INFO    MainThread:2767 [jupyter.py:save_ipynb():284] not saving jupyter notebook
2021-07-26 18:19:41,054 INFO    MainThread:2767 [wandb_init.py:_resume_backend():259] resuming backend
2021-07-26 18:19:41,056 INFO    MainThread:2767 [wandb_init.py:_pause_backend():250] pausing backend
2021-07-26 18:19:41,056 INFO    MainThread:2767 [jupyter.py:save_ipynb():284] not saving jupyter notebook
2021-07-26 18:24:14,451 INFO    MainThread:2767 [wandb_init.py:_resume_backend():259] resuming backend
2021-07-26 18:24:14,452 INFO    MainThread:2767 [wandb_init.py:_pause_backend():250] pausing backend
2021-07-26 18:24:14,453 INFO    MainThread:2767 [jupyter.py:save_ipynb():284] not saving jupyter notebook
2021-07-26 18:27:53,297 INFO    MainThread:2767 [wandb_init.py:_resume_backend():259] resuming backend
2021-07-26 18:27:53,480 INFO    MainThread:2767 [wandb_init.py:_pause_backend():250] pausing backend
2021-07-26 18:27:53,480 INFO    MainThread:2767 [jupyter.py:save_ipynb():284] not saving jupyter notebook
2021-07-26 18:29:06,431 INFO    MainThread:2767 [wandb_init.py:_resume_backend():259] resuming backend
2021-07-26 18:29:06,434 INFO    MainThread:2767 [pyagent.py:run():321] Starting sweep agent: entity=None, project=None, count=None
2021-07-26 18:29:07,220 INFO    Thread-13 :2767 [wandb_setup.py:_flush():69] setting env: {'project': 'tess', 'entity': 'parsellsx', 'root_dir': '/home/parsellsx/berkeley-seti', 'sweep_id': 't58vqmhu', 'run_id': 'l4nsom8z', 'sweep_param_path': '/home/parsellsx/berkeley-seti/wandb/sweep-t58vqmhu/config-l4nsom8z.yaml'}
2021-07-26 18:29:07,221 INFO    Thread-13 :2767 [wandb_init.py:_log_setup():337] Logging user logs to /home/parsellsx/berkeley-seti/wandb/run-20210726_182907-l4nsom8z/logs/debug.log
2021-07-26 18:29:07,221 INFO    Thread-13 :2767 [wandb_init.py:_log_setup():338] Logging internal logs to /home/parsellsx/berkeley-seti/wandb/run-20210726_182907-l4nsom8z/logs/debug-internal.log
2021-07-26 18:29:07,221 INFO    Thread-13 :2767 [wandb_init.py:init():370] calling init triggers
2021-07-26 18:29:07,221 INFO    Thread-13 :2767 [wandb_init.py:init():377] wandb.init called with sweep_config: {'C': 0.001, 'gamma': 'scale', 'kernel': 'poly', 'tol': 0.0001}
config: {}
2021-07-26 18:29:07,222 INFO    Thread-13 :2767 [wandb_init.py:init():419] starting backend
2021-07-26 18:29:07,222 INFO    Thread-13 :2767 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-07-26 18:29:07,225 INFO    Thread-13 :2767 [backend.py:ensure_launched():135] starting backend process...
2021-07-26 18:29:07,252 INFO    Thread-13 :2767 [backend.py:ensure_launched():140] started backend process with pid: 3739
2021-07-26 18:29:07,255 INFO    Thread-13 :2767 [wandb_init.py:init():424] backend started and connected
2021-07-26 18:29:07,287 INFO    Thread-13 :2767 [wandb_run.py:_config_callback():872] config_cb None None {'C': 0.001, 'gamma': 'scale', 'kernel': 'poly', 'tol': 0.0001}
2021-07-26 18:29:07,288 INFO    Thread-13 :2767 [wandb_run.py:_label_probe_notebook():844] probe notebook
2021-07-26 18:29:07,288 INFO    Thread-13 :2767 [wandb_run.py:_label_probe_notebook():851] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2021-07-26 18:29:07,288 INFO    Thread-13 :2767 [wandb_init.py:init():472] updated telemetry
2021-07-26 18:29:07,289 INFO    Thread-13 :2767 [wandb_init.py:init():491] communicating current version
2021-07-26 18:29:07,929 INFO    Thread-13 :2767 [wandb_init.py:init():496] got version response 
2021-07-26 18:29:07,929 INFO    Thread-13 :2767 [wandb_init.py:init():504] communicating run to backend with 30 second timeout
2021-07-26 18:29:08,034 INFO    Thread-13 :2767 [wandb_init.py:init():529] starting run threads in backend
2021-07-26 18:29:13,044 INFO    Thread-13 :2767 [wandb_run.py:_console_start():1623] atexit reg
2021-07-26 18:29:13,045 INFO    Thread-13 :2767 [wandb_run.py:_redirect():1497] redirect: SettingsConsole.WRAP
2021-07-26 18:29:13,045 INFO    Thread-13 :2767 [wandb_run.py:_redirect():1534] Wrapping output streams.
2021-07-26 18:29:13,436 INFO    Thread-13 :2767 [wandb_run.py:_redirect():1558] Redirects installed.
2021-07-26 18:29:13,437 INFO    Thread-13 :2767 [wandb_init.py:init():554] run started, returning control to user process
2021-07-26 18:29:14,239 INFO    Thread-13 :2767 [wandb_run.py:finish():1293] finishing run parsellsx/tess/l4nsom8z
2021-07-26 18:29:14,240 INFO    Thread-13 :2767 [wandb_run.py:_atexit_cleanup():1593] got exitcode: 1
2021-07-26 18:29:14,241 INFO    Thread-13 :2767 [wandb_run.py:_restore():1565] restore
2021-07-26 18:29:14,946 INFO    Thread-13 :2767 [wandb_run.py:_wait_for_finish():1715] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 842
  total_bytes: 842
}

2021-07-26 18:29:15,062 INFO    Thread-13 :2767 [wandb_run.py:_wait_for_finish():1715] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 842
  total_bytes: 842
}

2021-07-26 18:29:16,064 INFO    Thread-13 :2767 [wandb_run.py:_wait_for_finish():1715] got exit ret: file_counts {
  wandb_count: 2
}
pusher_stats {
  uploaded_bytes: 842
  total_bytes: 1277
}

2021-07-26 18:29:16,166 INFO    Thread-13 :2767 [wandb_run.py:_wait_for_finish():1715] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 842
  total_bytes: 18474
}

2021-07-26 18:29:16,269 INFO    Thread-13 :2767 [wandb_run.py:_wait_for_finish():1715] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 18474
  total_bytes: 18474
}

2021-07-26 18:29:16,372 INFO    Thread-13 :2767 [wandb_run.py:_wait_for_finish():1715] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 18474
  total_bytes: 18474
}

2021-07-26 18:29:16,474 INFO    Thread-13 :2767 [wandb_run.py:_wait_for_finish():1715] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 18474
  total_bytes: 18474
}

2021-07-26 18:29:16,576 INFO    Thread-13 :2767 [wandb_run.py:_wait_for_finish():1715] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 18474
  total_bytes: 18474
}

2021-07-26 18:29:16,731 INFO    Thread-13 :2767 [wandb_run.py:_wait_for_finish():1715] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 18474
  total_bytes: 18474
}

2021-07-26 18:29:16,833 INFO    Thread-13 :2767 [wandb_run.py:_wait_for_finish():1715] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 18474
  total_bytes: 18474
}

2021-07-26 18:29:17,941 INFO    Thread-13 :2767 [wandb_run.py:_show_files():1937] logging synced files
2021-07-26 18:29:17,948 ERROR   MainThread:2767 [pyagent.py:_run_jobs_from_queue():231] Run l4nsom8z errored: ValueError("could not convert string to float: 'EB'")
2021-07-26 18:29:18,282 INFO    Thread-22 :2767 [wandb_setup.py:_flush():69] setting env: {'project': 'tess', 'entity': 'parsellsx', 'root_dir': '/home/parsellsx/berkeley-seti', 'run_id': 'unb6fngi', 'sweep_param_path': '/home/parsellsx/berkeley-seti/wandb/sweep-t58vqmhu/config-unb6fngi.yaml', 'sweep_id': 't58vqmhu'}
2021-07-26 18:29:18,283 INFO    Thread-22 :2767 [wandb_setup.py:_flush():69] setting login settings: {}
2021-07-26 18:29:18,283 INFO    Thread-22 :2767 [wandb_init.py:_log_setup():337] Logging user logs to /home/parsellsx/berkeley-seti/wandb/run-20210726_182918-unb6fngi/logs/debug.log
2021-07-26 18:29:18,283 INFO    Thread-22 :2767 [wandb_init.py:_log_setup():338] Logging internal logs to /home/parsellsx/berkeley-seti/wandb/run-20210726_182918-unb6fngi/logs/debug-internal.log
2021-07-26 18:29:18,284 INFO    Thread-22 :2767 [wandb_init.py:init():370] calling init triggers
2021-07-26 18:29:18,284 INFO    Thread-22 :2767 [wandb_init.py:init():377] wandb.init called with sweep_config: {'C': 0.001, 'gamma': 'scale', 'kernel': 'poly', 'tol': 0.001}
config: {}
2021-07-26 18:29:18,284 INFO    Thread-22 :2767 [wandb_init.py:init():419] starting backend
2021-07-26 18:29:18,284 INFO    Thread-22 :2767 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-07-26 18:29:18,287 INFO    Thread-22 :2767 [backend.py:ensure_launched():135] starting backend process...
2021-07-26 18:29:18,294 INFO    Thread-22 :2767 [backend.py:ensure_launched():140] started backend process with pid: 3778
2021-07-26 18:29:18,295 INFO    Thread-22 :2767 [wandb_init.py:init():424] backend started and connected
2021-07-26 18:29:18,324 INFO    Thread-22 :2767 [wandb_run.py:_config_callback():872] config_cb None None {'C': 0.001, 'gamma': 'scale', 'kernel': 'poly', 'tol': 0.001}
2021-07-26 18:29:18,325 INFO    Thread-22 :2767 [wandb_run.py:_label_probe_notebook():844] probe notebook
2021-07-26 18:29:18,325 INFO    Thread-22 :2767 [wandb_run.py:_label_probe_notebook():851] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2021-07-26 18:29:18,325 INFO    Thread-22 :2767 [wandb_init.py:init():472] updated telemetry
2021-07-26 18:29:18,325 INFO    Thread-22 :2767 [wandb_init.py:init():491] communicating current version
2021-07-26 18:29:18,953 INFO    Thread-22 :2767 [wandb_init.py:init():496] got version response 
2021-07-26 18:29:18,953 INFO    Thread-22 :2767 [wandb_init.py:init():504] communicating run to backend with 30 second timeout
2021-07-26 18:29:19,050 INFO    Thread-22 :2767 [wandb_init.py:init():529] starting run threads in backend
2021-07-26 18:29:24,061 INFO    Thread-22 :2767 [wandb_run.py:_console_start():1623] atexit reg
2021-07-26 18:29:24,061 INFO    Thread-22 :2767 [wandb_run.py:_redirect():1497] redirect: SettingsConsole.WRAP
2021-07-26 18:29:24,061 INFO    Thread-22 :2767 [wandb_run.py:_redirect():1534] Wrapping output streams.
2021-07-26 18:29:24,062 INFO    Thread-22 :2767 [wandb_run.py:_redirect():1558] Redirects installed.
2021-07-26 18:29:24,062 INFO    Thread-22 :2767 [wandb_init.py:init():554] run started, returning control to user process
2021-07-26 18:29:24,889 INFO    Thread-22 :2767 [wandb_run.py:finish():1293] finishing run parsellsx/tess/unb6fngi
2021-07-26 18:29:24,889 INFO    Thread-22 :2767 [wandb_run.py:_atexit_cleanup():1593] got exitcode: 1
2021-07-26 18:29:24,889 INFO    Thread-22 :2767 [wandb_run.py:_restore():1565] restore
2021-07-26 18:29:25,570 INFO    Thread-22 :2767 [wandb_run.py:_wait_for_finish():1715] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 842
  total_bytes: 842
}

2021-07-26 18:29:25,688 INFO    Thread-22 :2767 [wandb_run.py:_wait_for_finish():1715] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 842
  total_bytes: 842
}

2021-07-26 18:29:26,073 INFO    Thread-22 :2767 [wandb_run.py:_wait_for_finish():1715] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 842
  total_bytes: 842
}

2021-07-26 18:29:26,175 INFO    Thread-22 :2767 [wandb_run.py:_wait_for_finish():1715] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 842
  total_bytes: 18473
}

2021-07-26 18:29:26,278 INFO    Thread-22 :2767 [wandb_run.py:_wait_for_finish():1715] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 18473
  total_bytes: 18473
}

2021-07-26 18:29:26,381 INFO    Thread-22 :2767 [wandb_run.py:_wait_for_finish():1715] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 18473
  total_bytes: 18473
}

2021-07-26 18:29:26,482 INFO    Thread-22 :2767 [wandb_run.py:_wait_for_finish():1715] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 18473
  total_bytes: 18473
}

2021-07-26 18:29:26,584 INFO    Thread-22 :2767 [wandb_run.py:_wait_for_finish():1715] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 18473
  total_bytes: 18473
}

2021-07-26 18:29:26,728 INFO    Thread-22 :2767 [wandb_run.py:_wait_for_finish():1715] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 18473
  total_bytes: 18473
}

2021-07-26 18:29:26,830 INFO    Thread-22 :2767 [wandb_run.py:_wait_for_finish():1715] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 18473
  total_bytes: 18473
}

2021-07-26 18:29:27,945 INFO    Thread-22 :2767 [wandb_run.py:_show_files():1937] logging synced files
2021-07-26 18:29:27,953 ERROR   MainThread:2767 [pyagent.py:_run_jobs_from_queue():231] Run unb6fngi errored: ValueError("could not convert string to float: 'EB'")
2021-07-26 18:29:28,247 INFO    Thread-31 :2767 [wandb_setup.py:_flush():69] setting env: {'project': 'tess', 'entity': 'parsellsx', 'root_dir': '/home/parsellsx/berkeley-seti', 'run_id': 'uc03mayt', 'sweep_param_path': '/home/parsellsx/berkeley-seti/wandb/sweep-t58vqmhu/config-uc03mayt.yaml', 'sweep_id': 't58vqmhu'}
2021-07-26 18:29:28,247 INFO    Thread-31 :2767 [wandb_setup.py:_flush():69] setting login settings: {}
2021-07-26 18:29:28,247 INFO    Thread-31 :2767 [wandb_init.py:_log_setup():337] Logging user logs to /home/parsellsx/berkeley-seti/wandb/run-20210726_182928-uc03mayt/logs/debug.log
2021-07-26 18:29:28,247 INFO    Thread-31 :2767 [wandb_init.py:_log_setup():338] Logging internal logs to /home/parsellsx/berkeley-seti/wandb/run-20210726_182928-uc03mayt/logs/debug-internal.log
2021-07-26 18:29:28,248 INFO    Thread-31 :2767 [wandb_init.py:init():370] calling init triggers
2021-07-26 18:29:28,248 INFO    Thread-31 :2767 [wandb_init.py:init():377] wandb.init called with sweep_config: {'C': 0.001, 'gamma': 'scale', 'kernel': 'poly', 'tol': 0.01}
config: {}
2021-07-26 18:29:28,248 INFO    Thread-31 :2767 [wandb_init.py:init():419] starting backend
2021-07-26 18:29:28,248 INFO    Thread-31 :2767 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-07-26 18:29:28,251 INFO    Thread-31 :2767 [backend.py:ensure_launched():135] starting backend process...
2021-07-26 18:29:28,258 INFO    Thread-31 :2767 [backend.py:ensure_launched():140] started backend process with pid: 3822
2021-07-26 18:29:28,259 INFO    Thread-31 :2767 [wandb_init.py:init():424] backend started and connected
2021-07-26 18:29:28,287 INFO    Thread-31 :2767 [wandb_run.py:_config_callback():872] config_cb None None {'C': 0.001, 'gamma': 'scale', 'kernel': 'poly', 'tol': 0.01}
2021-07-26 18:29:28,288 INFO    Thread-31 :2767 [wandb_run.py:_label_probe_notebook():844] probe notebook
2021-07-26 18:29:28,288 INFO    Thread-31 :2767 [wandb_run.py:_label_probe_notebook():851] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2021-07-26 18:29:28,288 INFO    Thread-31 :2767 [wandb_init.py:init():472] updated telemetry
2021-07-26 18:29:28,288 INFO    Thread-31 :2767 [wandb_init.py:init():491] communicating current version
2021-07-26 18:29:28,339 INFO    MainThread:2767 [wandb_init.py:_pause_backend():250] pausing backend
2021-07-26 18:29:28,339 INFO    MainThread:2767 [jupyter.py:save_ipynb():284] not saving jupyter notebook
2021-07-26 18:29:33,302 ERROR   Thread-31 :2767 [wandb_init.py:init():787] error
Traceback (most recent call last):
  File "/opt/conda/lib/python3.7/site-packages/wandb/sdk/wandb_init.py", line 761, in init
    run = wi.init()
  File "/opt/conda/lib/python3.7/site-packages/wandb/sdk/wandb_init.py", line 493, in init
    current_version=wandb.__version__
  File "/opt/conda/lib/python3.7/site-packages/wandb/sdk/interface/interface.py", line 796, in communicate_check_version
    result = self._communicate(rec)
  File "/opt/conda/lib/python3.7/site-packages/wandb/sdk/interface/interface.py", line 544, in _communicate
    return self._communicate_async(rec, local=local).get(timeout=timeout)
  File "/opt/conda/lib/python3.7/site-packages/wandb/sdk/interface/interface.py", line 82, in get
    is_set = self._object_ready.wait(timeout)
  File "/opt/conda/lib/python3.7/threading.py", line 552, in wait
    signaled = self._cond.wait(timeout)
  File "/opt/conda/lib/python3.7/threading.py", line 300, in wait
    gotit = waiter.acquire(True, timeout)
Exception
2021-07-26 18:43:02,038 INFO    MainThread:2767 [wandb_init.py:_resume_backend():259] resuming backend
2021-07-26 18:43:02,039 INFO    MainThread:2767 [wandb_settings.py:_apply_source_login():716] setting login settings: {}
2021-07-26 18:43:02,047 INFO    MainThread:2767 [pyagent.py:run():321] Starting sweep agent: entity=None, project=tess, count=None
2021-07-26 18:43:02,119 ERROR   MainThread:2767 [internal_api.py:execute():133] 404 response executing GraphQL.
2021-07-26 18:43:02,119 ERROR   MainThread:2767 [internal_api.py:execute():134] {"errors":[{"message":"could not find sweep parsellsx/tess/t58vqmhu during createAgent","path":["createAgent"]}],"data":{"createAgent":null}}
2021-07-26 18:43:02,234 INFO    MainThread:2767 [wandb_init.py:_pause_backend():250] pausing backend
2021-07-26 18:43:02,234 INFO    MainThread:2767 [jupyter.py:save_ipynb():284] not saving jupyter notebook
2021-07-26 18:43:27,809 INFO    MainThread:2767 [wandb_init.py:_resume_backend():259] resuming backend
2021-07-26 18:43:27,811 INFO    MainThread:2767 [wandb_settings.py:_apply_source_login():716] setting login settings: {}
2021-07-26 18:43:28,045 INFO    MainThread:2767 [wandb_init.py:_pause_backend():250] pausing backend
2021-07-26 18:43:28,045 INFO    MainThread:2767 [jupyter.py:save_ipynb():284] not saving jupyter notebook
2021-07-26 18:43:29,279 INFO    MainThread:2767 [wandb_init.py:_resume_backend():259] resuming backend
2021-07-26 18:43:29,281 INFO    MainThread:2767 [wandb_settings.py:_apply_source_login():716] setting login settings: {}
2021-07-26 18:43:29,289 INFO    MainThread:2767 [pyagent.py:run():321] Starting sweep agent: entity=None, project=tess, count=None
2021-07-26 18:43:29,668 INFO    Thread-35 :2767 [wandb_setup.py:_flush():69] setting env: {'project': 'tess', 'entity': 'parsellsx', 'root_dir': '/home/parsellsx/berkeley-seti', 'sweep_id': 'kbdrp5d7', 'run_id': 'joq21pye', 'sweep_param_path': '/home/parsellsx/berkeley-seti/wandb/sweep-kbdrp5d7/config-joq21pye.yaml'}
2021-07-26 18:43:29,668 INFO    Thread-35 :2767 [wandb_setup.py:_flush():69] setting login settings: {}
2021-07-26 18:43:29,668 INFO    Thread-35 :2767 [wandb_init.py:_log_setup():337] Logging user logs to /home/parsellsx/berkeley-seti/wandb/run-20210726_184329-joq21pye/logs/debug.log
2021-07-26 18:43:29,669 INFO    Thread-35 :2767 [wandb_init.py:_log_setup():338] Logging internal logs to /home/parsellsx/berkeley-seti/wandb/run-20210726_184329-joq21pye/logs/debug-internal.log
2021-07-26 18:43:29,669 INFO    Thread-35 :2767 [wandb_init.py:init():370] calling init triggers
2021-07-26 18:43:29,670 INFO    Thread-35 :2767 [wandb_init.py:init():377] wandb.init called with sweep_config: {'C': 0.001, 'gamma': 'scale', 'kernel': 'poly', 'tol': 0.0001}
config: {}
2021-07-26 18:43:29,670 INFO    Thread-35 :2767 [wandb_init.py:init():419] starting backend
2021-07-26 18:43:29,670 INFO    Thread-35 :2767 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-07-26 18:43:29,673 INFO    Thread-35 :2767 [backend.py:ensure_launched():135] starting backend process...
2021-07-26 18:43:29,680 INFO    Thread-35 :2767 [backend.py:ensure_launched():140] started backend process with pid: 3902
2021-07-26 18:43:29,682 INFO    Thread-35 :2767 [wandb_init.py:init():424] backend started and connected
2021-07-26 18:43:29,710 INFO    Thread-35 :2767 [wandb_run.py:_config_callback():872] config_cb None None {'C': 0.001, 'gamma': 'scale', 'kernel': 'poly', 'tol': 0.0001}
2021-07-26 18:43:29,711 INFO    Thread-35 :2767 [wandb_run.py:_label_probe_notebook():844] probe notebook
2021-07-26 18:43:29,711 INFO    Thread-35 :2767 [wandb_run.py:_label_probe_notebook():851] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2021-07-26 18:43:29,711 INFO    Thread-35 :2767 [wandb_init.py:init():472] updated telemetry
2021-07-26 18:43:29,716 INFO    Thread-35 :2767 [wandb_init.py:init():491] communicating current version
2021-07-26 18:43:30,340 INFO    Thread-35 :2767 [wandb_init.py:init():496] got version response 
2021-07-26 18:43:30,340 INFO    Thread-35 :2767 [wandb_init.py:init():504] communicating run to backend with 30 second timeout
2021-07-26 18:43:30,438 INFO    Thread-35 :2767 [wandb_init.py:init():529] starting run threads in backend
2021-07-26 18:43:35,462 INFO    Thread-35 :2767 [wandb_run.py:_console_start():1623] atexit reg
2021-07-26 18:43:35,463 INFO    Thread-35 :2767 [wandb_run.py:_redirect():1497] redirect: SettingsConsole.WRAP
2021-07-26 18:43:35,463 INFO    Thread-35 :2767 [wandb_run.py:_redirect():1534] Wrapping output streams.
2021-07-26 18:43:35,465 INFO    Thread-35 :2767 [wandb_run.py:_redirect():1558] Redirects installed.
2021-07-26 18:43:35,466 INFO    Thread-35 :2767 [wandb_init.py:init():554] run started, returning control to user process
2021-07-26 18:43:36,335 INFO    Thread-35 :2767 [wandb_run.py:finish():1293] finishing run parsellsx/tess/joq21pye
2021-07-26 18:43:36,335 INFO    Thread-35 :2767 [wandb_run.py:_atexit_cleanup():1593] got exitcode: 1
2021-07-26 18:43:36,336 INFO    Thread-35 :2767 [wandb_run.py:_restore():1565] restore
2021-07-26 18:43:36,971 INFO    Thread-35 :2767 [wandb_run.py:_wait_for_finish():1715] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 842
  total_bytes: 842
}

2021-07-26 18:43:37,449 INFO    Thread-35 :2767 [wandb_run.py:_wait_for_finish():1715] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 842
  total_bytes: 18474
}

2021-07-26 18:43:37,553 INFO    Thread-35 :2767 [wandb_run.py:_wait_for_finish():1715] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 842
  total_bytes: 18474
}

2021-07-26 18:43:37,655 INFO    Thread-35 :2767 [wandb_run.py:_wait_for_finish():1715] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 18474
  total_bytes: 18474
}

2021-07-26 18:43:37,758 INFO    Thread-35 :2767 [wandb_run.py:_wait_for_finish():1715] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 18474
  total_bytes: 18474
}

2021-07-26 18:43:37,860 INFO    Thread-35 :2767 [wandb_run.py:_wait_for_finish():1715] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 18474
  total_bytes: 18474
}

2021-07-26 18:43:37,962 INFO    Thread-35 :2767 [wandb_run.py:_wait_for_finish():1715] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 18474
  total_bytes: 18474
}

2021-07-26 18:43:38,174 INFO    Thread-35 :2767 [wandb_run.py:_wait_for_finish():1715] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 18474
  total_bytes: 18474
}

2021-07-26 18:43:38,276 INFO    Thread-35 :2767 [wandb_run.py:_wait_for_finish():1715] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 18474
  total_bytes: 18474
}

2021-07-26 18:43:39,398 INFO    Thread-35 :2767 [wandb_run.py:_show_files():1937] logging synced files
2021-07-26 18:43:39,409 ERROR   MainThread:2767 [pyagent.py:_run_jobs_from_queue():231] Run joq21pye errored: ValueError("could not convert string to float: 'EB'")
2021-07-26 18:43:39,735 INFO    Thread-44 :2767 [wandb_setup.py:_flush():69] setting env: {'project': 'tess', 'entity': 'parsellsx', 'root_dir': '/home/parsellsx/berkeley-seti', 'run_id': 'cifz6cir', 'sweep_param_path': '/home/parsellsx/berkeley-seti/wandb/sweep-kbdrp5d7/config-cifz6cir.yaml', 'sweep_id': 'kbdrp5d7'}
2021-07-26 18:43:39,735 INFO    Thread-44 :2767 [wandb_setup.py:_flush():69] setting login settings: {}
2021-07-26 18:43:39,735 INFO    Thread-44 :2767 [wandb_init.py:_log_setup():337] Logging user logs to /home/parsellsx/berkeley-seti/wandb/run-20210726_184339-cifz6cir/logs/debug.log
2021-07-26 18:43:39,735 INFO    Thread-44 :2767 [wandb_init.py:_log_setup():338] Logging internal logs to /home/parsellsx/berkeley-seti/wandb/run-20210726_184339-cifz6cir/logs/debug-internal.log
2021-07-26 18:43:39,736 INFO    Thread-44 :2767 [wandb_init.py:init():370] calling init triggers
2021-07-26 18:43:39,736 INFO    Thread-44 :2767 [wandb_init.py:init():377] wandb.init called with sweep_config: {'C': 0.001, 'gamma': 'scale', 'kernel': 'poly', 'tol': 0.001}
config: {}
2021-07-26 18:43:39,736 INFO    Thread-44 :2767 [wandb_init.py:init():419] starting backend
2021-07-26 18:43:39,736 INFO    Thread-44 :2767 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-07-26 18:43:39,739 INFO    Thread-44 :2767 [backend.py:ensure_launched():135] starting backend process...
2021-07-26 18:43:39,745 INFO    Thread-44 :2767 [backend.py:ensure_launched():140] started backend process with pid: 3943
2021-07-26 18:43:39,747 INFO    Thread-44 :2767 [wandb_init.py:init():424] backend started and connected
2021-07-26 18:43:39,777 INFO    Thread-44 :2767 [wandb_run.py:_config_callback():872] config_cb None None {'C': 0.001, 'gamma': 'scale', 'kernel': 'poly', 'tol': 0.001}
2021-07-26 18:43:39,777 INFO    Thread-44 :2767 [wandb_run.py:_label_probe_notebook():844] probe notebook
2021-07-26 18:43:39,778 INFO    Thread-44 :2767 [wandb_run.py:_label_probe_notebook():851] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2021-07-26 18:43:39,778 INFO    Thread-44 :2767 [wandb_init.py:init():472] updated telemetry
2021-07-26 18:43:39,778 INFO    Thread-44 :2767 [wandb_init.py:init():491] communicating current version
2021-07-26 18:43:40,411 INFO    Thread-44 :2767 [wandb_init.py:init():496] got version response 
2021-07-26 18:43:40,411 INFO    Thread-44 :2767 [wandb_init.py:init():504] communicating run to backend with 30 second timeout
2021-07-26 18:43:40,514 INFO    Thread-44 :2767 [wandb_init.py:init():529] starting run threads in backend
2021-07-26 18:43:41,935 INFO    MainThread:2767 [wandb_init.py:_pause_backend():250] pausing backend
2021-07-26 18:43:41,935 INFO    MainThread:2767 [jupyter.py:save_ipynb():284] not saving jupyter notebook
2021-07-26 18:43:42,105 ERROR   Thread-44 :2767 [wandb_init.py:init():787] error
Traceback (most recent call last):
  File "/opt/conda/lib/python3.7/site-packages/wandb/sdk/wandb_init.py", line 761, in init
    run = wi.init()
  File "/opt/conda/lib/python3.7/site-packages/wandb/sdk/wandb_init.py", line 532, in init
    _ = backend.interface.communicate_run_start(run_obj)
  File "/opt/conda/lib/python3.7/site-packages/wandb/sdk/interface/interface.py", line 806, in communicate_run_start
    result = self._communicate(rec)
  File "/opt/conda/lib/python3.7/site-packages/wandb/sdk/interface/interface.py", line 544, in _communicate
    return self._communicate_async(rec, local=local).get(timeout=timeout)
  File "/opt/conda/lib/python3.7/site-packages/wandb/sdk/interface/interface.py", line 82, in get
    is_set = self._object_ready.wait(timeout)
  File "/opt/conda/lib/python3.7/threading.py", line 552, in wait
    signaled = self._cond.wait(timeout)
  File "/opt/conda/lib/python3.7/threading.py", line 300, in wait
    gotit = waiter.acquire(True, timeout)
Exception
2021-07-26 18:58:56,930 INFO    MainThread:2767 [wandb_init.py:_resume_backend():259] resuming backend
2021-07-26 18:58:56,948 INFO    MainThread:2767 [wandb_init.py:_pause_backend():250] pausing backend
2021-07-26 18:58:56,949 INFO    MainThread:2767 [jupyter.py:save_ipynb():284] not saving jupyter notebook
2021-07-26 18:59:41,591 INFO    MainThread:2767 [wandb_init.py:_resume_backend():259] resuming backend
2021-07-26 18:59:41,613 INFO    MainThread:2767 [wandb_init.py:_pause_backend():250] pausing backend
2021-07-26 18:59:41,613 INFO    MainThread:2767 [jupyter.py:save_ipynb():284] not saving jupyter notebook
2021-07-26 18:59:49,609 INFO    MainThread:2767 [wandb_init.py:_resume_backend():259] resuming backend
2021-07-26 18:59:49,626 INFO    MainThread:2767 [wandb_init.py:_pause_backend():250] pausing backend
2021-07-26 18:59:49,626 INFO    MainThread:2767 [jupyter.py:save_ipynb():284] not saving jupyter notebook
2021-07-26 18:59:57,719 INFO    MainThread:2767 [wandb_init.py:_resume_backend():259] resuming backend
2021-07-26 18:59:57,733 INFO    MainThread:2767 [wandb_init.py:_pause_backend():250] pausing backend
2021-07-26 18:59:57,734 INFO    MainThread:2767 [jupyter.py:save_ipynb():284] not saving jupyter notebook
2021-07-26 19:10:39,595 INFO    MainThread:2767 [wandb_init.py:_resume_backend():259] resuming backend
2021-07-26 19:10:39,604 INFO    MainThread:2767 [wandb_init.py:_pause_backend():250] pausing backend
2021-07-26 19:10:39,605 INFO    MainThread:2767 [jupyter.py:save_ipynb():284] not saving jupyter notebook
2021-07-26 19:17:39,397 INFO    MainThread:2767 [wandb_init.py:_resume_backend():259] resuming backend
2021-07-26 19:17:39,404 INFO    MainThread:2767 [wandb_init.py:_pause_backend():250] pausing backend
2021-07-26 19:17:39,405 INFO    MainThread:2767 [jupyter.py:save_ipynb():284] not saving jupyter notebook
2021-07-26 19:17:51,349 INFO    MainThread:2767 [wandb_init.py:_resume_backend():259] resuming backend
2021-07-26 19:17:51,357 INFO    MainThread:2767 [wandb_settings.py:_apply_source_login():716] setting login settings: {}
2021-07-26 19:17:51,365 INFO    MainThread:2767 [wandb_init.py:_pause_backend():250] pausing backend
2021-07-26 19:17:51,365 INFO    MainThread:2767 [jupyter.py:save_ipynb():284] not saving jupyter notebook
2021-07-26 19:18:03,737 INFO    MainThread:2767 [wandb_init.py:_resume_backend():259] resuming backend
2021-07-26 19:18:03,746 INFO    MainThread:2767 [wandb_init.py:_pause_backend():250] pausing backend
2021-07-26 19:18:03,746 INFO    MainThread:2767 [jupyter.py:save_ipynb():284] not saving jupyter notebook
2021-07-26 19:19:08,860 INFO    MainThread:2767 [wandb_init.py:_resume_backend():259] resuming backend
2021-07-26 19:19:08,874 INFO    MainThread:2767 [wandb_settings.py:_apply_source_login():716] setting login settings: {}
2021-07-26 19:19:08,884 INFO    MainThread:2767 [wandb_setup.py:_flush():69] setting env: {'project': 'tess', 'entity': 'parsellsx', 'root_dir': '/home/parsellsx/berkeley-seti', 'run_id': 'cifz6cir', 'sweep_param_path': '/home/parsellsx/berkeley-seti/wandb/sweep-kbdrp5d7/config-cifz6cir.yaml', 'sweep_id': 'kbdrp5d7'}
2021-07-26 19:19:08,884 INFO    MainThread:2767 [wandb_setup.py:_flush():69] setting login settings: {}
2021-07-26 19:19:08,884 INFO    MainThread:2767 [wandb_init.py:_log_setup():337] Logging user logs to /home/parsellsx/berkeley-seti/wandb/run-20210726_191908-cifz6cir/logs/debug.log
2021-07-26 19:19:08,884 INFO    MainThread:2767 [wandb_init.py:_log_setup():338] Logging internal logs to /home/parsellsx/berkeley-seti/wandb/run-20210726_191908-cifz6cir/logs/debug-internal.log
2021-07-26 19:19:08,885 INFO    MainThread:2767 [wandb_init.py:init():370] calling init triggers
2021-07-26 19:19:08,885 INFO    MainThread:2767 [wandb_init.py:init():377] wandb.init called with sweep_config: {'C': 0.001, 'gamma': 'scale', 'kernel': 'poly', 'tol': 0.001}
config: {}
2021-07-26 19:19:08,885 INFO    MainThread:2767 [wandb_init.py:init():419] starting backend
2021-07-26 19:19:08,885 INFO    MainThread:2767 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-07-26 19:19:08,888 INFO    MainThread:2767 [backend.py:ensure_launched():135] starting backend process...
2021-07-26 19:19:08,895 INFO    MainThread:2767 [backend.py:ensure_launched():140] started backend process with pid: 4159
2021-07-26 19:19:08,896 INFO    MainThread:2767 [wandb_init.py:init():424] backend started and connected
2021-07-26 19:19:08,926 INFO    MainThread:2767 [wandb_run.py:_config_callback():872] config_cb None None {'C': 0.001, 'gamma': 'scale', 'kernel': 'poly', 'tol': 0.001}
2021-07-26 19:19:08,926 INFO    MainThread:2767 [wandb_run.py:_label_probe_notebook():844] probe notebook
2021-07-26 19:19:08,927 INFO    MainThread:2767 [wandb_run.py:_label_probe_notebook():851] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2021-07-26 19:19:08,927 INFO    MainThread:2767 [wandb_init.py:init():472] updated telemetry
2021-07-26 19:19:08,927 INFO    MainThread:2767 [wandb_init.py:init():491] communicating current version
2021-07-26 19:19:09,571 INFO    MainThread:2767 [wandb_init.py:init():496] got version response 
2021-07-26 19:19:09,571 INFO    MainThread:2767 [wandb_init.py:init():504] communicating run to backend with 30 second timeout
2021-07-26 19:19:22,847 WARNING MainThread:2767 [wandb_init.py:init():781] interrupted
Traceback (most recent call last):
  File "/opt/conda/lib/python3.7/site-packages/wandb/sdk/wandb_init.py", line 761, in init
    run = wi.init()
  File "/opt/conda/lib/python3.7/site-packages/wandb/sdk/wandb_init.py", line 505, in init
    ret = backend.interface.communicate_run(run, timeout=30)
  File "/opt/conda/lib/python3.7/site-packages/wandb/sdk/interface/interface.py", line 666, in communicate_run
    return self._communicate_run(run, timeout=timeout)
  File "/opt/conda/lib/python3.7/site-packages/wandb/sdk/interface/interface.py", line 654, in _communicate_run
    resp = self._communicate(req, timeout=timeout)
  File "/opt/conda/lib/python3.7/site-packages/wandb/sdk/interface/interface.py", line 544, in _communicate
    return self._communicate_async(rec, local=local).get(timeout=timeout)
  File "/opt/conda/lib/python3.7/site-packages/wandb/sdk/interface/interface.py", line 82, in get
    is_set = self._object_ready.wait(timeout)
  File "/opt/conda/lib/python3.7/threading.py", line 552, in wait
    signaled = self._cond.wait(timeout)
  File "/opt/conda/lib/python3.7/threading.py", line 300, in wait
    gotit = waiter.acquire(True, timeout)
KeyboardInterrupt
2021-07-26 19:19:22,874 INFO    MainThread:2767 [wandb_init.py:_pause_backend():250] pausing backend
2021-07-26 19:19:22,874 INFO    MainThread:2767 [jupyter.py:save_ipynb():284] not saving jupyter notebook
2021-07-26 19:19:22,986 INFO    MainThread:2767 [wandb_run.py:_atexit_cleanup():1593] got exitcode: 0
2021-07-26 19:19:22,986 INFO    MainThread:2767 [wandb_run.py:_restore():1565] restore
2021-07-26 19:19:22,988 INFO    MainThread:2767 [wandb_run.py:_restore():1565] restore
